### PR TITLE
tweak format

### PR DESCRIPTION
**Pull request by mfahampshire**
This change is [Reviewable](https://reviewable.io/reviews/nymtech/nym/5295)

### CODE REVIEW

File: documentation/docs/components/outputs/command-outputs/nym-node-migrate-help.md [hunk 1]
```diff
@@ -1,72 +1,8 @@
 ```sh
```
Comment (serinko) [inline documentation/docs/components/outputs/command-outputs/nym-node-migrate-help.md:1]: I think that the latest release may not even allow to migrate, that's why this came. out empty

File: documentation/docs/components/outputs/command-outputs/nym-node-migrate-help.md [hunk 2]
```diff
@@ -1,72 +1,8 @@
...
-      --enforce-zk-nyms <ENFORCE_ZK_NYMS>
-          Indicates whether this gateway is accepting only coconut credentials for accessing the mixnet or if it also accepts non-paying clients [env: NYMNODE_ENFORCE_ZK_NYMS=] [possible values: true, false]
-      --mnemonic <MNEMONIC>
-          Custom cosmos wallet mnemonic used for zk-nym redemption. If no value is provided, a fresh mnemonic is going to be generated [env: NYMNODE_MNEMONIC=]
-      --upstream-exit-policy-url <UPSTREAM_EXIT_POLICY_URL>
-          Specifies the url for an upstream source of the exit policy used by this node [env: NYMNODE_UPSTREAM_EXIT_POLICY=]
-      --open-proxy <OPEN_PROXY>
-          Specifies whether this exit node should run in 'open-proxy' mode and thus would attempt to resolve **ANY** request it receives [env: NYMNODE_OPEN_PROXY=] [possible values: true, false]
-  -h, --help
-          Print help
```
Comment (serinko) [inline documentation/docs/components/outputs/command-outputs/nym-node-migrate-help.md:71]: this should be left there and note that operators first have to migrate to pre crunch and then upgrade. I can do the last bit if you want.

File: documentation/docs/components/outputs/command-outputs/nym-node-migrate.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+```sh
+** Attention **
```
Comment (serinko) [inline documentation/docs/components/outputs/command-outputs/nym-node-migrate.md:2]: Here we go. . that's crunch message re migration.

File: documentation/docs/components/outputs/command-outputs/nym-node-node-details.md [hunk 4]
```diff
@@ -1,3 +1,3 @@
 ```sh
-[2m2024-11-06T12:18:46.742880Z[0m [31mERROR[0m [2mnym-node/src/config/upgrade_helpers.rs[0m[2m:[0m[2m17[0m[2m:[0m Failed to finish upgrade - failed to load config file using path '/home/m/.nym/nym-nodes/default-nym-node/config/config.toml'. detailed message: No such file or directory (os error 2)
+[2m2024-12-18T17:36:27.019221Z[0m [31mERROR[0m [2mnym-node/src/config/upgrade_helpers.rs[0m[2m:[0m[2m19[0m[2m:[0m Failed to finish upgrade - failed to load config file using path '/home/m/.nym/nym-nodes/default-nym-node/config/config.toml'. detailed message: No such file or directory (os error 2)
```
Comment (serinko) [inline documentation/docs/components/outputs/command-outputs/nym-node-node-details.md:2]: this seems like a bad outcome

**Review by serinko — CHANGES_REQUESTED** [review]
I can help you on the operators page later today if you want. There has been some changes on the migration side of things.

File: documentation/docs/components/outputs/command-outputs/nym-node-node-details.md (hunk 4 above)
Comment (mfahampshire) [reply documentation/docs/components/outputs/command-outputs/nym-node-node-details.md:2]: We aren't using this right now I don't think? I need to fix this at some point though..

File: documentation/docs/components/outputs/command-outputs/nym-node-migrate.md (hunk 3 above)
Comment (mfahampshire) [reply documentation/docs/components/outputs/command-outputs/nym-node-migrate.md:2]: So I remove it?

File: documentation/docs/components/outputs/command-outputs/nym-node-migrate-help.md (hunk 2 above)
Comment (mfahampshire) [reply documentation/docs/components/outputs/command-outputs/nym-node-migrate-help.md:71]: if you dont mind that'd be great

**Review by serinko — APPROVED** [review]

**Pull request merged**